server/views/patterns/typography.njk

```diff
@@ -123,257 +121,254 @@ p q r s t u v w x y z { | } ~
   </div>
 
   <h3>Figures</h3>
-  <p>Figures are usually used to refer to images:</p>
+
+  <h4>Image-based figure</h4>
   <div class="example">
     <figure>
-      <img src="http://lorempixel.com/680/400/abstract" alt="Example image" />
+      <img style="max-width: 100%;"
```

Comment:
Preach!

But I think we need more than *just* an e.g. image-with-caption to start building components – we're essentially treading water a bit until we get something a bit more sured-up out of C&T. Which will hopefully be any day soon.

This also largely answers the 'why' of this PR – essentially just tidying up/removing cruft for want of something more substantial to be getting our teeth in to.